### DISCUSSION TITLE
Corporation of police that work in Taupo !

### DISCUSSION THREAD

**Discussion by Mactengaming** [base/web#1123] [General]
The corporation of police department that is in Taupo.Look I took these jokers to the cleaners once again .On the 5th of December 2023 .I was on trial against the police .I walked away because I was protected by my tipuna that spiritually not connected cannot see .I am From Aotearoa

**Comment by Mactengaming** [comment]
Did I mention that now they have been harassing me .I dont know if y'all know about the declaration my tipuna yes I whakapapa back anyone that is mareikura does .We have been treated un lawfully.Now I'm going to stand in my strength and take them again

**Reply by sky-coderay** [reply]
))

**Reply by MEGLDON83** [reply]
(;)

**Reply by valerkaaa** [reply]
cool
